website/common/script/index.js

```diff
@@ -298,7 +298,6 @@ api.wrap = function wrapUser (user, main = true) {
     handleTwoHanded: _.partial(importedFns.handleTwoHanded, user),
     predictableRandom: _.partial(importedFns.predictableRandom, user),
     crit: _.partial(importedFns.crit, user),
-    randomVal: _.partial(importedFns.randomVal, user),
```

Comment:
Because the user is no longer passed in. `randomVal` should probably be moved to the libs folder, since it is no longer a user function.